### PR TITLE
expand staff info for edit

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: src/components/profile/edit/EditPersonalInfo.vue [hunk 1]
```diff
@@ -180,27 +176,51 @@
...
-            :metaValue="staffInformation.wprDeskNumber.value"
-          />
-          <Meta
-            metaKey="Cost centre"
-            :metaValue="staffInformation.costCenter.value"
-          />
-        </MetaList>
-      </div>
+      <label for="field-worker-type">Worker Type</label>
+      <input type="text" disabled :value="staffInformation.workerType.value" />
```
Comment (hidde) [inline src/components/profile/edit/EditPersonalInfo.vue]: this one lacks `id="field-worker-type"`

File: src/components/profile/edit/EditPersonalInfo.vue [hunk 2]
```diff
@@ -180,27 +176,51 @@
...
+        label="Worker type privacy levels"
+        id="field-worker-type-privacy"
+        profileFieldName="staffInformation.workerType"
+        :profileFieldObject="staffInformation.workerType"
+        :disabled="true"
+      />
+
+      <hr role="presentation" />
+
+      <label for="field-office-location">Cost Center</label>
```
Comment (hidde) [inline src/components/profile/edit/EditPersonalInfo.vue]: this for attribute doesn't correspond to an input with that ID

File: src/components/profile/edit/EditPersonalInfo.vue [hunk 3]
```diff
@@ -180,27 +176,51 @@
...
+      <input type="text" disabled :value="staffInformation.costCenter.value" />
```
Comment (hidde) [inline src/components/profile/edit/EditPersonalInfo.vue]: `id` missing

File: src/components/profile/edit/EditPersonalInfo.vue [hunk 4]
```diff
@@ -180,27 +176,51 @@
...
+        label="Cost center privacy levels"
+        id="field-cost-center-privacy"
+        profileFieldName="staffInformation.costCenter"
+        :profileFieldObject="staffInformation.costCenter"
+        :disabled="true"
+      />
+
+      <hr role="presentation" />
+
+      <label for="field-office-location">Desk Number</label>
```
Comment (hidde) [inline src/components/profile/edit/EditPersonalInfo.vue]: the `for` does not correspond with an id on the input

File: src/components/profile/edit/EditPersonalInfo.vue [hunk 5]
```diff
@@ -180,27 +176,51 @@
...
+        label="Desk numnber privacy levels"
+        id="field-desk-number-privacy"
+        profileFieldName="staffInformation.wprDeskNumber"
+        :profileFieldObject="staffInformation.wprDeskNumber"
+        :disabled="true"
+      />
+
+      <hr role="presentation" />
+
+      <label for="field-office-location">Team</label>
```
Comment (hidde) [inline src/components/profile/edit/EditPersonalInfo.vue]: same as above

**Comment by fiji-flo** [comment]
copied from office location which had `id` missing, too :P

**Pull request merged**